frontend/src/Themes.jsx

```diff
@@ -141,7 +141,7 @@ export const DataScrollView = styled.SafeAreaView`
     justify-content: center;
 `;
 
-export const LightingTitle = styled(PageTitle)`
+export const AmbianceTitle = styled(PageTitle)`
```

Comment:
It's not a big deal but making changes like this should be taken with care. Because these are changes that don't have anything to do with the feature/patch of this branch, but more importantly is that they're also hidden in a completely unrelated commit, without any mention in the commit message. If one was to go back through the git history at this point, there would be no mention of these name changes and other developers might never see that these changes.

Reply:
ok, that makes sense, I'll make sure to put those in separate commits or do theme changes in separate branches/PRs.